docs/releases/1.11.txt

```diff
@@ -171,8 +171,9 @@ Minor features
   :class:`~django.contrib.postgres.aggregates.StringAgg` determines if
   concatenated values will be distinct.
 
-* The new :class:`~django.contrib.postgres.indexes.GinIndex` class allows
-  creating gin indexes in the database.
+* The new :class:`~django.contrib.postgres.indexes.GinIndex`  and
+  :class:`~django.contrib.postgres.indexes.BrinIndex` classes allow
+  creating gin and ``brin`` indexes in the database.
```

Comment:
The spelling of `gin` and `brin` was debated a bit in this PR already. It seems reasonably to use the same spelling as in the manual. Is changing gin -> GIN in this commit OK?